### DISCUSSION TITLE
Double-clicking in the application does not execute the method “onDoubleTap”

### DISCUSSION THREAD

**Discussion by mxdlsm** [Genymobile/feedback#90] [Genymotion Desktop] [answered]
The Android application will be executed by double clicking on other emulators or real machines, but it will not be executed on genymotion and there will be no reaction. How to solve this problem?

![CleanShot 2024-04-11 at 18 03 00@2x](https://github.com/Genymobile/feedback/assets/22886455/b2de75f6-14e6-48bb-b5e0-738d233d0621)

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @mxdlsm,
We are sorry but this discussion board is strictly reserved to feature requests and suggestions. It it not appropriate for technical assistance requests.
Please visit our help center at https://support.genymotion.com/hc/en-us/ instead.
Thank you for your understanding.
